utils/nctl/sh/scenarios/emergency_upgrade_test_balances.sh

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/utils/main.sh
+source "$NCTL"/sh/views/utils.sh
+source "$NCTL"/sh/assets/upgrade.sh
+source "$NCTL"/sh/scenarios/common/itst.sh
+source "$NCTL"/sh/node/svc_"$NCTL_DAEMON_TYPE".sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that performs an emergency restart on the network.
+# It also simulates social consensus on replacing the original validators (nodes 1-5)
+# with a completely new set (nodes 6-10).
```

Comment:
This comment is incorrect.  The test only ever runs nodes 1-5 as the validator set and doesn't try to change that via emergency upgrade.